### DISCUSSION TITLE
Nasal administration in MoBi

### DISCUSSION THREAD

**Discussion by Nathalie06** [Open-Systems-Pharmacology/Forum#763] [Questions & Problems] [answered]
Hi, 

I am a new PK-sim user and I want to add a possible intra nasal administration for a compound. I tried in PK-sim using a mixt between infusion and oral administration but I can't describe the asbsorption, especially the Cmax. 
I think I have to use a more mechanistic model with a part of the dose absorbed by lungs, a part through the gastro intestinal tract and a part lost by degradation or clearance. To do this I have to use MoBi.
I have seen another post [https://github.com/Open-Systems-Pharmacology/MoBi/issues/246] but for the first MoBi use it is not easy.
Is there a public model using this administration route ? or is it possible to share a model ?

Thanks in advance,
Best regards,
Nathalie

**Comment by StephanSchaller** [comment, marked as answer]
Dear Nathalie,

I am not aware of any ready-to use published model for MoBi for this purpose, but it has been about 1 year since I did a thorough check in literature.

Best,
Stephan
